publish_template.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# Push a new Inferno version to inferno-template GitHub Repository via SSH.
+#
+# USAGE:
+#   ./publish_template
+
+set -e
+set -x
+
+
+REPO_URL=${1:-git@github.com:inferno-framework/inferno-template.git}
+VERSION=$(bundle exec ./bin/inferno version)
+
+
+if [ -d ./inferno-template ]; then
+  echo "ERROR: detected inferno-template directory, please remove or rename it to run this script."
+  exit 1;
+fi;
+
+git clone $REPO_URL ./inferno-template
+rm -f ./inferno-template/Gemfile.lock
+
+bundle exec ./bin/inferno new inferno-template --author "Inferno Template" --force
+
+git -C ./inferno-template add --all
+git -C ./inferno-template commit --message "update inferno-template to v$VERSION"
+git -C ./inferno-template push origin main
```

Comment:
This should open a new branch and let the user click on the output of this command in the CLI to open a PR (which is provided by the git command, i think?) and then they can manually review the changes as a sanity check.  Pushing to main from an automated script is very scary to me.  We should protect `main` anyhow, just haven't really gotten to it because it is updated so infrequently.